<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="DESKTOP-4OUNIVO" name="test.TestCardSpace" tests="5" failures="0" timestamp="27 Apr 2018 16:36:18 GMT" time="0.001" errors="1">
  <testcase name="testCardSprite" time="0.000" classname="test.TestCardSpace">
    <skipped/>
  </testcase> <!-- testCardSprite -->
  <testcase name="testGetName" time="0.000" classname="test.TestCardSpace">
    <skipped/>
  </testcase> <!-- testGetName -->
  <testcase name="testCardCnt" time="0.000" classname="test.TestCardSpace">
    <skipped/>
  </testcase> <!-- testCardCnt -->
  <testcase name="testDrawCard" time="0.000" classname="test.TestCardSpace">
    <skipped/>
  </testcase> <!-- testDrawCard -->
  <testcase name="setUp" time="0.001" classname="test.TestCardSpace">
    <error type="java.io.FileNotFoundException" message="\resources\config\chance.json (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: \resources\config\chance.json (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.CardSpace.parseConfig(Unknown Source)
at core.CardSpace.<init>(Unknown Source)
at test.TestCardSpace.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
    </error>
  </testcase> <!-- setUp -->
</testsuite> <!-- test.TestCardSpace -->
